### DISCUSSION TITLE
strapi v5

### DISCUSSION THREAD

**Discussion by runsense** [Fekide/strapi-plugin-translate#451] [Q&A] [answered]
hi, in web site creation, 
i wish used srtapi-plugin-translate for
automated translate.

but give me 
design-system error in first
with a lot around 
translate fact..

exist-il a way for doing this.

thank you

**Comment by sargreal** [comment, marked as answer]
Hi @runsense,

we are still in progress of migrating to v5. Please follow issue #404 for updates.
